frontend/packages/dev-console/src/components/GuidedTourAddAction.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { getGuidedToursWithStatus } from '@console/app/src/components/guided-tours/utils/guided-tour-utils';
+import GuidedTourTile from './GuidedTourTile';
+
+const GuidedTourAddAction: React.FC = () => {
```

Comment:
Why exactly do we need this extra component here? It doesn't seem to be doing anything at the moment.

Reply:
It shows/hides GuidedTourTile based on the availabilty of the tours

Reply:
I've removed this